README.md

```diff
@@ -320,6 +322,23 @@ if erl_triggered // quota left in the quotaKey bucket
 if !erl_triggered // ERL wasn't triggered in this call, so we haven't identified the remaining quota.
 ```
 
+### Use of fixed window on Take and TakeElevated
+If you want to use fixed window on Take or TakeElevated, you can do so by setting the `fixed_window` property in the bucket configuration to `true` (default `false`). This will refill the bucket at the specified interval instead of granular.
+
+On top of that, you can use the `fixed_window` property in the `configOverride` parameter to safely activate/deactivate the new fixed_window algorithm from the client on-demand.
```

Comment:
What would be the benefit or use case of being able to enable this on demand? What would happen if clients are doing requests with fixed window on and off all the time?

Reply:
> What would be the benefit or use case of being able to enable this on demand?
We would be able to do a controlled rollout. Nowadays everyone is using sliding window; we want to introduce the fixed window algorithm but wanna make sure the transition is controlled. If something doesn't work as expected sending the fixed_window=false (feature flag) param will fallback to sliding window

> What would happen if clients are doing requests with fixed window on and off all the time?
That shouldn't happen. It's the same risk for clients calling take with and without overrides.